pkg/utils/printer/printer.go

```diff
@@ -169,6 +197,24 @@ func (p *TextPrinter) Print(v interface{}) error {
 	return nil
 }
 
+func (p *TextPrinter) Info(i interface{}) {
+	flag := viper.GetBool(config.ArgVerbose)
+	if flag {
+		fmt.Print(i)
+	} else {
+		return
+	}
+}
+
+func (p *TextPrinter) Infof(format string, a ...interface{}) {
+	flag := viper.GetBool(config.ArgVerbose)
+	if flag {
+		fmt.Printf("[INFOF] "+format+"\n", a...)
+	} else {
+		return
+	}
+}
```

Comment:
Same duplicate code here - you could reuse `Info()` from above.

Reply:
removed

Reply:
done